x-pack/plugins/security/server/routes/api_keys/query_aggregations.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RouteDefinitionParams } from '..';
+import type { ApiKeyAggregationsResponse } from '../../../common/model';
+import { wrapIntoCustomErrorResponse } from '../../errors';
+import { createLicensedRouteHandler } from '../licensed_route_handler';
+
+export function defineQueryApiKeysAggregationsRoute({
```

Comment:
__question:__ I see our UI always hits these two endpoints. Is there any use case where we'd want to call just one of these API endpoints? If not, would it make sense to join them and return results in a single response?